doc/development.md

```diff
@@ -118,39 +118,49 @@ When an experimental version of the Ignition config spec (e.g.: `3.1.0-experimen
 - The HTTP `Accept` header that Ignition uses whenever fetching a config will be updated to advertise the new stable spec.
 - New features will be documented in the [migrating configs](doc/migrating-configs.md) documentation.
 
-The code changes that are required to achieve these effects are typically the following:
+The changes that are required to achieve these effects are typically the following:
 
 ### Making the experimental package stable
 
 - Rename `config/vX_Y_experimental` to `config/vX_Y`, and update the golang `package` statements
-- Update `MaxVersion` in `config/vX_Y/types/config.go` to have `PreRelease` set to `""`
+- Drop `_experimental` from all imports in `config/vX_Y`
+- Update `MaxVersion` in `config/vX_Y/types/config.go` to delete the `PreRelease` field
 - Update `config/vX_Y/config_test.go` to test that the new stable version is valid and the old experimental version is invalid
 - Update the `Accept` header in `internal/resource/url.go` to specify the new spec version.
 
 ### Creating the new experimental package
 
 - Copy `config/vX_Y` into `config/vX_(Y+1)_experimental`, and update the golang `package` statements
+- Update all `config/vX_Y` imports in `config/vX_(Y+1)_experimental` to `config/vX_(Y+1)_experimental`
 - Update `MaxVersion` in `config/vX_(Y+1)_experimental/types/config.go` to have the correct major/minor versions and `PreRelease` set to `"experimental"`
-- Update `config/vX_(Y+1)_experimental/config.go` to use `config/vX_Y` for parsing
-- Update `config/vX_(Y+1)_experimental/config_test.go` to test that the new stable version is valid, the new experimental version is valid, and the old experimental version is invalid
-- Update `config/vX_(Y+1)_experimental/translate/translate.go` to only set the new version.
-- Update `config/config.go` to handle the new config.
+- Update `config/vX_(Y+1)_experimental/config_test.go` to test that the new stable version is invalid and the new experimental version is valid
+- Update `config/vX_(Y+1)_experimental/translate/translate.go` to translate from the previous stable version.  Update the `old_types` import, delete all functions except `translateIgnition` and `Translate`, and ensure `translateIgnition` translates the entire `Ignition` struct.
+- Update `config/config.go` to handle the new stable and experimental versions.
```

Comment:
Should this be split into two lines, one in `Making the experimental package stable` for adding the new stable version and then leave only the new experimental version portion here?

Reply:
Could do, but I don't think it's worth it.  The `config.go` changes are somewhat intertwined, and the code isn't expected to compile at this point anyway.